dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/TransactionHelper.java

```diff
@@ -34,7 +34,7 @@ public final class TransactionHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(TransactionHelper.class);
     private Session session;
-    private RuntimeException thrown;
+    private Throwable thrown;
```

Comment:
By the same token, the changes in this file seem to affect `/updateUserWorkflows` which should maybe be deleted anyway (we probably don't use it)